MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/labs/SpinningSymbolLayerIconActivity.java

```diff
@@ -0,0 +1,180 @@
+import static com.mapbox.mapboxsdk.style.layers.PropertyFactory.iconAllowOverlap;
+import static com.mapbox.mapboxsdk.style.layers.PropertyFactory.iconPitchAlignment;
+import static com.mapbox.mapboxsdk.style.layers.PropertyFactory.iconRotate;
+
+/**
+ * Use an Android-system ValueAnimator and the Maps SDK's runtime styling functionality to
+ * continually updated a SymbolLayer icons' rotation. This creates a spinning effect.
+ */
+public class SpinningSymbolLayerIconActivity extends AppCompatActivity implements OnMapReadyCallback {
+
+  private static final String SOURCE_ID = "SOURCE_ID";
+  private static final String ICON_ID = "ICON_ID";
+  private static final String LAYER_ID = "LAYER_ID";
+  private static final int DESIRED_SECONDS_PER_ONE_FULL_360_SPIN = 1;
+  private MapView mapView;
+  private MapboxMap mapboxMap;
+  private ValueAnimator iconSpinningAnimator;
+
+  @Override
+  protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+
+    // Mapbox access token is configured here. This needs to be called either in your application
+    // object or in the same activity which contains the mapview.
+    Mapbox.getInstance(this, getString(R.string.access_token));
+
+    // This contains the MapView in XML and needs to be called after the access token is configured.
+    setContentView(R.layout.activity_spinning_symbol_layer_icon);
+
+    // Initialize the MapView
+    mapView = findViewById(R.id.mapView);
+    mapView.onCreate(savedInstanceState);
+    mapView.getMapAsync(this);
+  }
+
+  @Override
+  public void onMapReady(final MapboxMap mapboxMap) {
+    this.mapboxMap = mapboxMap;
+
+    // Create a Style object
+    mapboxMap.setStyle(new Style.Builder().fromUri(Style.MAPBOX_STREETS)
+
+      // Add the SymbolLayer icon image to the map style
+      .withImage(ICON_ID, BitmapUtils.getBitmapFromDrawable(
+        getResources().getDrawable(R.drawable.hurricane_icon)))
+
+      // Adding a GeoJson source for the SymbolLayer icons.
+      .withSource(new GeoJsonSource(SOURCE_ID, loadGeoJsonFromAsset("spinning_icon.geojson")))
+
+      // Adding the actual SymbolLayer to the map style.
+      .withLayer(new SymbolLayer(LAYER_ID, SOURCE_ID)
+        .withProperties(PropertyFactory.iconImage(ICON_ID),
+          iconAllowOverlap(true))
+      ), new Style.OnStyleLoaded() {
+        @Override
+        public void onStyleLoaded(@NonNull Style style) {
+          startIconSpinningAnimation();
+        }
+      });
+  }
+
+  /**
+   * Set up and start the icon spinning animation. The Android system ValueAnimator emits a new value, which is
+   * used as the SymbolLayer icons' rotation value.
+   */
+  private void startIconSpinningAnimation() {
+    if (iconSpinningAnimator != null) {
+      iconSpinningAnimator.cancel();
+    }
+    iconSpinningAnimator = ValueAnimator.ofFloat(0, 360);
+    iconSpinningAnimator.setDuration(DESIRED_SECONDS_PER_ONE_FULL_360_SPIN * 1000);
```

Comment:
Per the name, the multiplication should be hardcoded into the variable itself.

Reply:
Fixed